bucket/tesseract.json

```diff
@@ -42,8 +43,8 @@
         "tesseract-languages": "tesseract-languages"
     },
     "checkver": {
-        "re": "tesseract-ocr-w32-setup-v(?<version>[\\d.]+).exe",
-        "url": "https://digi.bib.uni-mannheim.de/tesseract/"
+        "re": "tesseract-ocr-w32-setup-v(4.[\\d\\w.-]+)\\.exe",
```

Comment:
Is this hardcoded `4` needed? When you use C=M.. workaround it will always be on top

Reply:
Version 4 is the current stable version, and version 3 is previous release, it's not updated since version 4 stable released. In the near future, no hardcoded version number is needed.